packages/portfolio-contract/tools/plan-solve.ts

```diff
@@ -366,7 +366,8 @@ export const rebalanceMinCostFlowSteps = async (
    * Add 20% to a fee estimate in case the landscape changes between estimation
    * and execution.
    */
-  const padFeeEstimate = (estimate: bigint): bigint => (estimate * 120n) / 100n;
+  const padFeeEstimate = (estimate: bigint): bigint =>
+    estimate <= 0n ? estimate : (estimate * 120n - 1n) / 100n + 1n;
```

Comment:
changing the rounding is observable at runtime.

hence by definition not a refactor, in my book. hm.

Reply:
In a strict sense yes, although in practice I think our actual values are big and round enough to not be affected, and at any rate this switch from rounding down to rounding up _can't_ have an effect of more than one minor unit (1 ubld/1 gwei/etc.). But if you prefer, I can move this change an isolated "chore" PR.